pkg/xray/watches.go

```diff
@@ -0,0 +1,246 @@
+package xray
+
+import (
+	"context"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"log"
+	"net/http"
+
+	"github.com/go-resty/resty/v2"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+type WatchGeneralData struct {
+	Name        string `json:"name"`
+	Description string `json:"description,omitempty"`
+	Active      bool   `json:"active,omitempty"`
+}
+
+type WatchFilterValue struct {
+	Key   string `json:"key,omitempty"`
+	Value string `json:"value,omitempty"`
+}
+
+type WatchFilter struct {
+	Type  string `json:"type,omitempty"`
+	Value string `json:"value,omitempty"`
+}
+
+type WatchProjectResource struct {
+	Type            string        `json:"type,omitempty"`
+	BinaryManagerId string        `json:"bin_mgr_id,omitempty"`
+	Filters         []WatchFilter `json:"filters,omitempty"`
+	// Watch a repo
+	Name string `json:"name,omitempty"`
```

Comment:
Formatting here?

Reply:
All files were auto-formatted before committing. Looks like the comment in line 34 broke the code block into 2 pieces. Will fix it.